mewarpx/mewarpx/utils_store/util.py

```diff
@@ -387,3 +388,35 @@ def plasma_Debye_length(T, n):
     return np.sqrt(
         constants.epsilon_0 * constants.kb_J * T / (n * constants.e**2)
     )
+
+def calc_timesteps(V_anode, V_grid, CFL_factor, nx, nz, xmin, xmax, zmin, zmax):
```

Comment:
We should make it that `V_grid` is an optional parameter (just use default `None`) since not all simulations will have an auxiliary grid. If `V_grid` is not specified it's value can be set to `5` (this will give a safe value for `vmax` in the case that the electron velocities are only set by their temperature).